test/HeadersTest.php

```diff
@@ -647,4 +647,14 @@ public function testStrictKeyComparisonInGet(): void
         $headers = Mail\Headers::fromString("000: foo-bar");
         $this->assertFalse($headers->get('0'));
     }
+
+    /** @group issues-175 */
```

Comment:
```suggestion
    /** @group issue-175 */
```

makes more sense to use singular than plural